### ISSUE TITLE
Rider error when starting the IDE

### ISSUE THREAD

**Issue by iamfreee** [zielu/GitToolBox#46]
After updating the plugin to the most recent version:

```
Plugin 'zielu.gittoolbox' failed to initialize and will be disabled.  Please restart Rider.

java.lang.AbstractMethodError
    at com.intellij.openapi.components.impl.ComponentManagerImpl.registerComponentInstance(ComponentManagerImpl.java:417)
    at com.intellij.openapi.components.impl.ComponentManagerImpl.access$300(ComponentManagerImpl.java:58)
    at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentConfigComponentAdapter.getComponentInstance(ComponentManagerImpl.java:482)
    at com.intellij.openapi.components.impl.ComponentManagerImpl.createComponents(ComponentManagerImpl.java:120)
    at com.intellij.openapi.application.impl.ApplicationImpl.lambda$createComponents$8(ApplicationImpl.java:466)
    at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$1(CoreProgressManager.java:176)
    at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:556)
    at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:501)
    at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:66)
    at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:163)
    at com.intellij.openapi.application.impl.ApplicationImpl.createComponents(ApplicationImpl.java:473)
    at com.intellij.openapi.components.impl.ComponentManagerImpl.init(ComponentManagerImpl.java:104)
    at com.intellij.openapi.application.impl.ApplicationImpl.load(ApplicationImpl.java:425)
    at com.intellij.openapi.application.impl.ApplicationImpl.load(ApplicationImpl.java:411)
    at com.intellij.idea.IdeaApplication.run(IdeaApplication.java:203)
    at com.intellij.idea.MainImpl$1$1.lambda$proceed$0(MainImpl.java:63)
    at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
    at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:762)
    at java.awt.EventQueue.access$500(EventQueue.java:98)
    at java.awt.EventQueue$3.run(EventQueue.java:715)
    at java.awt.EventQueue$3.run(EventQueue.java:709)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:732)
    at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:343)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by zielu** [comment]
Hi, is there more ? I cannot see real cause here, but I'll try to install Rider and reproduce

**Comment by zielu** [comment]
You can rollback to 171.1.2 for now. I will need to have dedicated release for 171 platform (Rider) and other products (172)

1. download https://plugins.jetbrains.com/plugin/download?updateId=33639
2. Setting > Plugins > Install plugin from disk...

**Comment by iamfreee** [comment]
Thanks for the answer.

😄

**Comment by zielu** [comment]
Fixed in 171.3.0
